frontend/src/metabase/admin/permissions/selectors/data-permissions/group-sidebar.unit.spec.ts

```diff
@@ -122,6 +116,12 @@ describe("getGroupsDataPermissionEditor", () => {
         icon: "close",
         iconColor: "danger",
       },
+      {
+        label: `Granular`,
```

Comment:
This should be above No as granular is more permissive than completely no access.

Reply:
![CleanShot 2024-08-01 at 15 30 19@2x](https://github.com/user-attachments/assets/c88c748b-a0c4-4c31-97f1-42721a2a6eea)

Fixed